org.civicrm.reqangular/js/test/mocks/models/instances/contact-instance-mock.js

```diff
@@ -16,7 +16,7 @@ define([
        * @return {boolean}
        */
       isInstance: function (object) {
-        return _.isEqual(_.functions(object), _.functions(instance));
+        return !!_.intersection(_.functionsIn(object), _.functions(instance)).length;
```

Comment:
I don't think this code works the same as the previous one.

If `object` has methods a, b, and c, and `instance` has methods a and b:

• The previous code would return false.
• The new code would return true.

Is this the goal?

Reply:
you're right, i botched the original logic. Solution was pretty simple, use `functionsIn` on both object (`functions` on 3.x returned own and inherited properties, while in 4.x it has changed to return only own properties)